ext/pdo_dblib/tests/types.phpt

```diff
@@ -64,3 +69,5 @@ bool(true)
 bool(true)
 bool(true)
 bool(true)
+bool(true)
+bool(true)
```

Comment:
There are PDO attributes that are not implemented in pdo_dblib now: PDO::ATTR_SERVER_INFO, PDO::ATTR_SERVER_VERSION, PDO::ATTR_CLIENT_VERSION. They could be used to reveal some info about connection including TDS protocol version. But I think it is a task for a separate PR. I'll put a comment about possible fail in the test now with a TODO about these attrs.